_source/logzio_collections/_metrics-sources/docker.md

```diff
@@ -38,7 +39,7 @@ shipping-tags:
     logzio/docker-collector-metrics
```

Comment:
Please add `LOGZIO_MODULES="docker"` to the sample code

Reply:
Done.